### ISSUE TITLE
[Request] Add setting for exclusion list separate from "files.exclude"?

### ISSUE THREAD

**Issue by joezappie** [biati-digital/vscode-show-hide-excluded-files#1]
I use devcontainers which allows you to specify workspace settings separate from your `settings.json` in a file called `.devcontainer/devcontainer.json`.

Since its generally a bad idea to commit `settings.json` file and the fact that clicking the toggle button changes that file, theres no good way to commit your excluded files setup.

What I propose is to add another option that can be specified in the remote, workspace or user that can define this exclusion list to then generate the `settings.json` config.

```
"showHideExcludedConfig.exclude": {
   ".devcontainer": true,
    ".vscode": true,
    "**/node_modules": true,
    "**/.git": true,
    "**/build": true,
}
```

Basically this is a clone of `files.exclude` but is never changed when toggling so its safe to commit and is used as the default exclusion list. Upon vscode startup or when the button is clicked, it would add these items to the `files.exclude` list if they don't already exist. If they do exist, then it toggles them like normal.

**Comment by joezappie** [comment]
I'd be open to making a PR for this as well.

**Comment by biati-digital** [comment]
Hi, when we created the extension we did a few tests with different projects and came to the conclusion that adding a setting for this will not always work, users are forced to keep the same config for `showHideExcludedConfig.exclude` and `files.exclude` some projects already have a workspace settings.json with custom excludes that the extension should also be able to handle. This was months ago so i do not remember exactly our conclusion but i do remember that we noticed this problem because we handle multiple projects with multiple frameworks and configurations. 

Anyway, I'm open to a PR and if everything works great i'll submit an update.

**Comment by joezappie** [comment]
My thought on this was `showHideExcludedConfig.exclude` would be optional. Any parameters defined in `files.exclude` but not in `showHideExcludedConfig.exclude` should function as they do now. Thanks for the insight into your attempt at something similar. I'm interested in playing around with vscode extension development so I'll pull the project and see what I can come up with. 

I did find the extension "Tidy Explorer" which seems to do it similar to what I was thinking, but for me its to cumbersome. It lets you setup a bunch of different filters that you can individually toggle off, when I really love the simplicity of just a hide icon in the status bar like this extension provides!

**Comment by biati-digital** [comment]
Awesome, we also wanted something really simple so please check it and let me know if you have other ideas or something can be improved.

**Comment by joezappie** [comment]
Submitted a PR. Still a bit of work to be done but curious if this functionality makes sense to you. Added details in the PR description.
